InstanceExport/PowerShell/README.md

```diff
@@ -27,8 +27,63 @@ The process will prompt to you the mandatory fields to be typed in your screen `
 It will run automatically after you enter the fields and it will prompt the progress of your data been exported by each available category. When all is finished you should see the message "Exporting Instance finished".
 
 The created power shell script has five parameters that are explained below:
+
 - manifestFilePath: Path (or just file name, if same directory) of the manifest file you have downloaded using step 2, e.g. `C:\Users\MyUser\InstanceExport\manifest.json`.
 - exportFilePath: Path (or just file name, if same directory) to create a folder for the exported files, e.g. `C:\Users\MyUser\InstanceExport\2021_Export\`; you do not need to create this folder manually, the script will do so for you.
 - userName: Your username (work email) for login at DevResults, e.g. `first.last@org.org`.
 - password: Your password for login at DevResults or API Key's Secret.
 - overwrite: Optional parameter to inform if you want to overwrite files that already exist and replace them. If you don't use the value of it will be false, which means that if a file already exists in the exportFilePath it will be skipped.
+
+### Troubleshooting
+
+When running the InstanceExport script in your local machine you might see a few error messages that could indicate an issue with downloading the information from our servers and saving it to your choosen folder. The following error messages are examples of what kind of messages you can see in the Power Shell window that you've run the script and that also should be present in a log file created for each run of the export. In order to help you solve these issues we provide here a few possible solutions that can help you solve the issues and successfully extract the data you need.
+
+```
+An error occurred
+Error occurred when extracting data from /api/awards/123/documents/Project long name/Quarterly and Annual Report long name/Q04 Reporting submissions May - Jul 2019/Annual Workplan Progress Review/A_very_long_file_name_that_will_cause_exception_v1.xlsx
+StatusCode:
+Url: /api/awards/123/documents/Project long name/Quarterly and Annual Report long name/Q04 Reporting submissions May - Jul 2019/Annual Workplan Progress Review/A_very_long_file_name_that_will_cause_exception_v1.xlsx
+StatusDescription:
+```
+
+- This error message means that the Instance Export script was able to find the data on our servers but most probably something is wrong with the file path. For example, the path can be too long so PowerShell did not succeed on saving the file to your export folder. Our recommendation to solve the problem is : please try again to run PowerShell 7 (64 bit version) in your local machine so the path too long would not be a problem to download and be saved to local folder. You probably will need to rename the file after the export run if your are using Windows OS.
```

Comment:
I've made a few suggested edits here:
```suggestion
- This error message means that the Instance Export script was able to find the data on our servers but most probably something is wrong with the file path. For example, the path could be too long and PowerShell could not save the file to your export folder. Our recommendation to solve the problem is : please try to run the export again using PowerShell 7 (64 bit version) which can handle longer paths. You probably will need to rename the file after the export run if you are using Windows OS.. Alternatively, rename the file or parts of the folder path in DevResults to reduce the overall path length.
```

And here's the same edits but with a different format that might be easier for people to skim:
```suggestion
- This error message means that the Instance Export script was able to find the data on our servers but most probably something is wrong with the file path. For example, the path could be too long and PowerShell could not save the file to your export folder.  
    **Our recommendations:**  
    1. Please try to run the export again using PowerShell 7 (64 bit version) which can handle longer paths. You probably will need to rename the file after the export run if you are using Windows OS.  
    2. Alternatively, rename the file or parts of the folder path in DevResults to reduce the overall path length.
```